csharp/HOLMS.Platform/HOLMS.Platform/Application.Client/SessionService.cs

```diff
@@ -10,28 +9,44 @@ public class SessionService {
         public SessionContext SessionContext { get; protected set; }
         public string SessionUsername => SessionContext.UserName;
 
-        public bool SessionCan(AuthrnCapability c) {
+        public bool IsTokenAuthorized(SecurityAction e) {
             if (SessionContext.User == null) {
                 return false;
             }
 
-            switch (c) {
-                case AuthrnCapability.CreditCardSetup:
-                    return SessionContext.Capabilities.Contains(Capability.CreditCardSetup);
-                case AuthrnCapability.GroupManagement:
-                    return SessionContext.Capabilities.Contains(Capability.GroupManagement);
-                case AuthrnCapability.HousekeepingManagement:
-                    return SessionContext.Capabilities.Contains(Capability.HousekeepingManagement);
-                case AuthrnCapability.StaffManagement:
-                    return SessionContext.Capabilities.Contains(Capability.StaffManagement);
-                case AuthrnCapability.SystemSettings:
-                    return SessionContext.Capabilities.Contains(Capability.SystemSettings);
-                case AuthrnCapability.YieldManagement:
-                    return SessionContext.Capabilities.Contains(Capability.YieldManagement);
-                case AuthrnCapability.OverrideRates:
-                    return SessionContext.Capabilities.Contains(Capability.RateOverride);
+            switch (e) {
+                case SecurityAction.AdjustFolio:
+                    return false;
+                case SecurityAction.ChangeSystemSettings:
+                    return false;
+                case SecurityAction.GetRoomTypes:
+                    return false;
+                case SecurityAction.GetSupplySnapshot:
+                    return false;
+                case SecurityAction.ManageGroups:
+                    return false;
+                case SecurityAction.ManageGuests:
+                    return false;
+                case SecurityAction.ManageHousekeeping:
+                    return false;
+                case SecurityAction.ManageIncidentalReservation:
+                    return false;
+                case SecurityAction.ManageMoney:
+                    return false;
```

Comment:
Can you make these check and return the correct result, rather than just `false`?